contributing.md

```diff
@@ -0,0 +1,80 @@
+> # Meraki Frontend Repository
+
+This repository contains the source code for the Meraki Frontend. The Meraki Frontend is a web application that allows users to learn python , spoken english and other things.
+
+The Deployed Links For Meraki.
+
+[Meraki Production ]("https://merakilearn.org")
+
+[Meraki Development ]("https://bhanwari-devi.vercel.app")
+
+> ### Tech Stack
+
+- React
+- Redux
+- [MUI](https://mui.com/)
+- nodeJs (V14)
+
+> # Installation and Setup Guide
+
+1. Fork the repository
+2. create a Clone of the repository
+3. Make sure you have nodejs installed on your system with Version 14
```

Comment:
If the user has a current version of Node.js installed on their computer, I think they may be dissuaded from contributing if they're asked to downgrade it. We could have them use `nvm` or maybe we could add it as a dependency so they don't have to do this (https://stackoverflow.com/a/57965993).